setup/InstallerChecks/InstallerChecks/InstallerChecks.csproj

```diff
@@ -0,0 +1,56 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can we convert this into a VS2017 project?  We should be able to use inline packages and inline versions, if so.